### ISSUE TITLE
->parshaHebrew Show the wrong parsha next year!

### ISSUE THREAD

**Issue by StaffCollab** [zman-org/zman#33]
It shows next year ויקהל -פקודי together, but it will be separate.

**Comment by pintend** [comment]
Can you show code, it looks like its working

`\Zman\Zman::parse('3/9/2024')->parsha === 'Vayakheil'`

**Comment by StaffCollab** [comment]
@shrage-oberlander Please clarify.

Please check 2025, I spoke about next year

**Comment by pintend** [comment]
oh, i see. you are correct, it is wrong. I thought 2024 because you posted in 2023...


also https://github.com/zman-org/zman/issues/35

**Comment by zachweix** [comment]
I have worked on this project in the past, but you can check my more recent work at https://github.com/zachweix/PhpZmanim. I admit that I need to update the documentation, but if you are using a new enough version of the library you can get a date (make sure it's Shabbos, during the week it will return nothing) and get the parsha.

**Comment by pintend** [comment]
https://github.com/zman-org/zman/blob/ec4a4f5f4454ecbf394120b90c771a5cf3dc0b92/src/Getters/Parsha.php#L89
https://github.com/zman-org/zman/blob/ec4a4f5f4454ecbf394120b90c771a5cf3dc0b92/src/Getters/Parsha.php#L90

`$shabbos === 21` so `$this->assertEquals('Vayakheil', Zman::parse('03/22/2025')->parsha)` fails

**Comment by pintend** [comment]
Pull request https://github.com/zman-org/zman/pull/36

**Comment by yskraus** [comment]
what about next weeks parsha?
is it also included in this commit?

**Comment by pintend** [comment]
No, its caused by a different bug on line 135 as you mentioned.

the issue i have with fixing it, is there are no tests that fail when we comment out the code and removing it "fixes" it for this year

**Comment by pintend** [comment]
next week parsha is caused by https://github.com/zman-org/zman/blob/ec4a4f5f4454ecbf394120b90c771a5cf3dc0b92/src/Getters/Parsha.php#L135

all tests pass when that code is commented out and `Zman\Zman::parse('9/7/2024')->parsha === 'Shoftim'`

What case was that offset added for? before removing it we should have a test that fails...

**Comment by Shrage-Oberlander** [comment]
@pintend do you have a fixed version of the Vayakeil Pekidai issue for this year?
